package.json

```diff
@@ -19,6 +19,12 @@
   "bugs": {
     "url": "https://github.com/katowulf/mockfirebase/issues"
   },
+  "files": [
+    "*.md",
+    "src/",
+    "tutorials/",
+    "browser/"
```

Comment:
Yup, I agree with you on using `package.json/files` over `.npmignore`. I just wish Bower and npm chose to use the same format! npm's `"files"` versus Bower's `"ignore"` always trips me up.